packages/tools/src/drawingSvg/drawPolyline.ts

```diff
@@ -53,7 +53,7 @@ export default function drawPolyline(
   let pointsAttribute = '';
 
   for (const point of points) {
-    pointsAttribute += `${point[0]}, ${point[1]} `;
+    pointsAttribute += `${point[0].toFixed(1)}, ${point[1].toFixed(1)} `;
```

Comment:
maybe to 2? or you think 1 is enough

Reply:
This is in canvas pixels, so I really think 1 is sufficient.  2 would be overkill - we can't position at 1/100th of a pixel.  Of course that could be 1/4 pixel positioning with scaling, but even so I think it is ok.

Reply:
Fair enough

Reply:
@lscoder please perform this in your new Path element as well